src/Pages/MembershipApplication/MembershipForm.js

```diff
@@ -73,6 +74,31 @@ export default function MembershipForm (props) {
       })
   }
 
+  async function sendVerificationEmail () {
```

Comment:
To abstract axios requests, I put all API calls into `src/APIFunctions/`. Could you put this inside of a Profile.js file in the directory? See [Event.js as an example](https://github.com/SCE-Development/Core-v4/blob/dev/src/APIFunctions/Event.js)

Reply:
Did this is my newest push